packages/cli/src/bin/pipcook-plugin.ts

```diff
@@ -56,7 +114,12 @@ program
     }
     install(name, opts);
   });
-
+program
+  .command('installFromConfig <pipeline>')
```

Comment:
Hmm, I'm -1 on this command-line style, how about:

- `pipcook pipeline install <URL>` is to tell users, install the plugins from a pipeline with URL.

Reply:
Done.